Document/0x05a-Platform-Overview.md

```diff
@@ -643,3 +643,5 @@ The Android application attack surface consists of all components of the applica
   - [Data Storage on Android](0x05d-Testing-Data-Storage.md#data-storage-on-android "Data Storage on Android")
 - Protect itself against compromised environments, repackaging or other local attacks, see also:
   - [Android Anti-Reversing Defenses](0x05j-Testing-Resiliency-Against-Reverse-Engineering.md#android-anti-reversing-defenses "Android Anti-Reversing Defenses")
+- Implement the client side security measures such as Root Detection, Certificate/Public Key Pinning

```

Comment:
For me this is implied in the previous lines.

Reply:
Yes, but it is not explicitly mentioned. It should be there for new security enthusiasts.

Reply:
We understand your concern and thanks for pointing that out but we really believe this is already well covered, e.g. talking about "protecting against compromised environments", implies talking about root detection. The corresponding links will lead the reader to the desired information.